index.html

```diff
@@ -90,7 +90,234 @@ <h1 class="primary-text-header">I’m Steven Levoyer to see you!</h1>
+                                    <p class="text-p">Back End Dev</p>
+                                </div>
+                                <div class="counter"></div>
+                                <div class="year">
+                                    <p class="text-p">2015</p>
+                                </div>
+                            </div>
+                        </div>
+                        <div class="secondary-text-work">
+                            <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                        </div>
+                        <div class="tags">
+                            <ul class="list-tag">
+                                <li class="tag"><a href="#" class="text-s">html</a></li>
+                                <li class="tag"><a href="#" class="text-s">css</a></li>
+                                <li class="tag"><a href="#" class="text-s">javaScript</a></li>
+                            </ul>
+                        </div>
+                        <div class="action">
+                            <button type="button" class="action-button">
+                                <p class="text-a">See Project</p>
+                            </button>
+                        </div>
+                    </div>
+                </div>
+                <div class="card-works">
+                    <img class="snapshoot" src="./img/Snapshoot-3.svg">
+                    <div class="left-block">
+                        <div class="primary-text-work">
+                            <h2 class="project-title">Tonic</h2>
+                            <div class="frame-2">
+                                <div class="client">
+                                    <p class="text-p">CANOPY</p>
+                                </div>
+                                <div class="counter"></div>
+                                <div class="role">
+                                    <p class="text-p">Back End Dev</p>
+                                </div>
+                                <div class="counter"></div>
+                                <div class="year">
+                                    <p class="text-p">2015</p>
+                                </div>
+                            </div>
+                        </div>
+                        <div class="secondary-text-work">
+                            <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                        </div>
+                        <div class="tags">
+                            <ul class="list-tag">
+                                <li class="tag"><a href="#" class="text-s">html</a></li>
+                                <li class="tag"><a href="#" class="text-s">css</a></li>
+                                <li class="tag"><a href="#" class="text-s">javaScript</a></li>
+                            </ul>
```

Comment:
- [ ] Kindly note that your list of technologies is not displaying as it is on the figma template. Kindly correct this please.
### Your design
![1](https://user-images.githubusercontent.com/94551484/213792020-fcbfddce-b159-4f91-8d41-92252056817a.png)
### Figma
![f1](https://user-images.githubusercontent.com/94551484/213792026-b6b52b68-c5ec-411f-8da6-45900a6ad526.png)

Reply:
> Hi, @Kamba56 I have a question regarding this comment Do you have any idea why in my computer the tags are aligned in a row and in the image it is broken and placed in a column?

@mdg0410, I think you are not using the 375px screen size(which is the accepted phone screen size) ... Try resizing your screen to 375px and see.

Reply:
Thanks bro @Kamba56 
I already thought of a possible solution.
I will proceed to request another revision.
If you have time you would help me a lot by checking if I managed to correct that, on my pc it works and on yours, it works?

Good luck